code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm

```diff
@@ -549,3 +549,20 @@
 		return TRUE
 	else
 		return FALSE
+
+// ----------------------------
+//  Sci smartfridge
+// ----------------------------
+/obj/machinery/smartfridge/sci
+	desc = "A smart storage vender for tech."
+
+/obj/machinery/smartfridge/sci/accept_check(obj/item/O)
+	if(istype(O, /obj/item/stock_parts/))
+		return TRUE
+	if(istype(O, /obj/item/disk/tech_disk/))
+		return TRUE
+	if(istype(O, /obj/item/circuit_component/))
+		return TRUE
+	if(istype(O, /obj/item/assembly/))
```

Comment:
trailing `/` on typepaths, remove them

Also this should probably use a typecache, look for other examples of typecaches and use that

Reply:
like this?
/obj/machinery/smartfridge/sci/accept_check(obj/item/O)
	if(typecacheof(list(0, /obj/item/stock_parts, /obj/item/disk/tech_disk, /obj/item/circuit_component, /obj/item/assembly)))
		return TRUE
	return FALSE